### DISCUSSION TITLE
Bring back the old feed!

### DISCUSSION THREAD

**Discussion by mrjones2014** [community/community#66344] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I consider this a bug honestly. After [this feedback](https://github.com/orgs/community/discussions/13154) and many others like it, you decided to now _force_ the "For You" tab onto everyone? Really? Now there is NO WAY AT ALL to actually get just the information I want..... are you serious?????

Good god, GitHub is a tool, not a social media platform! Give me back my old, chronological, following-only feed!

![image](https://github.com/community/community/assets/8648891/41eeafd7-7c45-47a1-9d66-0c1240fee543)

**Comment by fdwr** [comment]
Clearly the people who pushed this misfeature do not actually *use* the GitHub home page, or else they'd know better. 😕

**Reply by leeoniya** [reply]
i think the engineers who implemented it and then rolled it out must actually use Gitlab internally and are actively trying to sabotage Github by doing the opposite of what users asked for. there's simply no other logical explanation for such a colossal failure.

good job, people! bring on the LinkedIn integration! 🤮

**Comment by isyuricunha** [comment]
I want GH to bring back the old feed.
